app/src/main/kotlin/com/x8bit/bitwarden/ui/vault/feature/itemlisting/util/VaultItemListingDataExtensions.kt

```diff
@@ -125,14 +125,16 @@ fun VaultData.toViewState(
 
     val filteredFailuresCipherViewList = decryptCipherListResult
         .failures
-        .map { cipher ->
+        .takeIf { autofillSelectionData == null }
```

Comment:
Can we do this filtering in `filterForAutofillIfNecessary` in the `VaultItemListingViewModel`?

We should be able to adjust the `vaultData.copy()` calls to set `failures = emptyList()`.

This probably needs to be applied to `filterForCredentialCreationIfNecessary` and `filterForTotpIfNecessary` also.